docs/content/concepts/partitions-schedules-sensors/partitions.mdx

```diff
@@ -133,6 +134,51 @@ result = materialize(
 )
 ```
 
+#### Dynamically partitioned assets (Experimental)
```

Comment:
```suggestion
#### Dynamically partitioned assets
```

I think `(Experimental)` was breaking the on page nav, so I'm suggesting we remove it. 🤷‍♀️